nengo/solvers.py

```diff
@@ -302,6 +314,28 @@ class LstsqDrop(Solver):
 
     This solver first solves for coefficients (decoders/weights) with
     L2 regularization, drops those nearest to zero, and retrains remaining.
+
+    Parameters
+    ----------
+    weights : bool, optional
+        If False, solve for decoders. If True, solve for weights.
+    drop : float, optional
+        Fraction of decoders or weights to set to zero.
+    solver1 : Solver, optional
+        Solver for finding the initial decoders.
+    solver2 : Solver, optional
+        Used for re-solving for the decoders after dropout.
+
+    Attributes
+    ----------
+    drop : float
+        Fraction of decoders or weights to set to zero.
+    solver1 : Solver
+        Solver for finding the initial decoders.
+    solver2 : Solver
+        Used for re-solving for the decoders after dropout.
+    weights : bool
+        If False, solve for decoders. If True, solve for weights.
```

Comment:
This is a general comment, but there's often a lot of redundancy between our "Parameters" and "Attributes" sections.

Reply:
Yeah for sure, it would be nice to reduce that duplication in a way that isn't too hard to read. Might be a thing to do when also moving the parameter stuff to a separate package? If we want to do that?

Reply:
I find the Attributes sections tend to get out of date a lot as well (since it's harder to line them up with the code, versus just looking at the function definition).

Reply:
Made #1558 to track that.